llvm/test/CodeGen/AMDGPU/live-interval-bug-in-rename-independent-subregs.mir

```diff
@@ -0,0 +1,5426 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+# RUN: llc -mtriple=amdgcn--amdpal -mcpu=gfx1010 -start-before=rename-independent-subregs -stop-before=amdgpu-mark-last-scratch-load %s -o - | FileCheck -check-prefix=GCN %s
+
+--- |
```

Comment:
This test needs to be reduced, preferably to something which doesn't require an IR or registers section and at most a handful of blocks